types/address_test.go

```diff
@@ -53,11 +57,19 @@ var invalidStrs = []string{
 
 func (s *addressTestSuite) testMarshal(original, res interface{}, marshal func() ([]byte, error), unmarshal func([]byte) error) {
 	bz, err := marshal()
-	s.Require().Nil(err)
-	s.Require().Nil(unmarshal(bz))
+	s.Require().NoError(err)
+	s.Require().NoError(unmarshal(bz))
 	s.Require().Equal(original, res)
 }
 
+func testMarshalYAML(t *testing.T, original, res interface{}, marshal func() (interface{}, error), unmarshal func([]byte) error) {
+	bz, err := marshal()
+
+	assert.NoError(t, err)
```

Comment:
nit: here and others: use `require.NoError` instead so that your test can stop early.

Reply:
Done